### PR TITLE
Mention delay & backspace behavior

### PR DESCRIPTION
**Pull request by dozoisch**

### CODE REVIEW

**Comment by trekare** [comment]
LGTM

File: src/simditor-mention.coffee [hunk 1]
```diff
@@ -49,6 +29,14 @@ class SimditorMention extends SimpleModule
 
     @items = []
 
+    @editor.keystroke.add('backspace', 'a', (e, $node) =>
```
Comment (gaurav-townsqd) [inline src/simditor-mention.coffee:32]: 👍

File: src/simditor-mention.coffee [hunk 2]
```diff
@@ -3,26 +3,6 @@
...
-        func.apply context, args
-      return
-
-    callNow = immediate and not timeout
-    clearTimeout timeout
-    timeout = setTimeout(later, wait)
-    func.apply context, args if callNow
-    return
-
-
```
Comment (gaurav-townsqd) [inline src/simditor-mention.coffee:25]: 👍

File: src/simditor-mention.coffee [hunk 3]
```diff
@@ -414,9 +400,27 @@ class SimditorMention extends SimpleModule
...
 
-  _onKeyUp: (e)->
+  getTimeoutDelay:(val) ->
+    # The more people type, the more likely they are to stop typing
+    # so we speed up the process a bit
+    if (val.length <= 1)
+      return 400
+    if (val.length == 2)
+      return 300
+    return 250
```
Comment (gaurav-townsqd) [inline src/simditor-mention.coffee:410]: 👍

**Review by gaurav-townsqd — APPROVED** [review]
This is amazing! 

LGTM 👍

**Pull request merged**